### ISSUE TITLE
When will Ubuntu 22.04 (aarch64) be supported? Can I use the Ubuntu 20.04 apt repository on Ubuntu 22.04? Is that kosher?

### ISSUE THREAD

**Issue by Danrancan** [mono/mono#21751]
The [official Download Mono website](https://www.mono-project.com/download/stable/#download-lin-ubuntu) lists apt repositories for Ubuntu, up until Ubuntu 20.04, which is only a year and a half from being unsupported. For those doing an early upgrade to Ubuntu 22.04 (an a Raspberry Pi 4 aarch64), I want to know if it's okay to use the official Ubuntu 20.04 Focal Apt repository on an Ubuntu 22.04 Jammy  installation? Is that okay? 

If it's not kosher or okay, and will break things, then when will an apt repository come out that supports Ubuntu 22.04 Jammy?

Please let me know. Thank.

**Comment by akoeplinger** [comment]
Yes you can use the 20.04 repository on 22.04, so far I don't think I've seen any issues reported regarding that.

**Comment by knocte** [comment]
@Danrancan hey, what Alex is saying is true, it works for me.

That being said, I would recommend to start to avoid relying on this repository, because the fact that it hasn't been updated to 22.04 in an "official way" could mean that it might go away soon.

The way to migrate out of this is to, of course, migrate to the new versions of .NET (e.g. .NET7, .NET8, etc). If you can't really migrate to that yet, you could still install mono from official Ubuntu repositories. The only disadvantage of the latter approach is that the version of mono is older and the `msbuild` package is not there (instead, you have xbuild). The disadvantage of using xbuild compared to msbuild is that it's a bit ancient and has some bugs and limitations, for example, it cannot build .NETStandard projects. However, Ubuntu is now shipping `dotnet` repositories out of the box (without the need to add `apt` repositories from Microsoft), so technically you could use `dotnet build` to build .NETStandard projects before using xbuild for your old .NET4.x mono-based projects. It's what we use in my company. If you want to know more, just ask away.

**Comment by neclimdul** [comment]
Time to start asking the same question for 24.04 too i guess.

I have a external tool relying on the "official" repo and so doesn't work on these versions of ubuntu.
